CMakeLists.txt

```diff
@@ -163,6 +163,7 @@ add_subdirectory(src/OrbitTest)
 add_subdirectory(src/OrbitVersion)
 add_subdirectory(src/PresetFile)
 add_subdirectory(src/Symbols)
+add_subdirectory(src/Tracing)
```

Comment:
Of course this comes directly from `LinuxTracing`, but could we have the name expanded a bit, e.g., to `TracingInterface`? `Tracing` and `orbit_tracing` alone feel too generic to me.

Reply:
Yeah it is a git generic. Changed to `TracingInterface` and `orbit_tracing_interface`